File: Photo Viewer/Photo Viewer/DetailViewController.m
//
//  DetailViewController.m
//  Photo Viewer
//
//  Created by Chang Tong Xue on 2015-12-16.
//  Copyright © 2015 DX. All rights reserved.
//

#import "DetailViewController.h"

@interface DetailViewController ()
@property (weak, nonatomic) IBOutlet UIImageView *logo;
@end

@implementation DetailViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    self.logo.image = [UIImage imageNamed:self.photo.fileName];
    self.navigationItem.title = self.photo.name;
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
}

-(void)touchesBegan:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
    [self performSegueWithIdentifier:@"info" sender:self];
}

#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    InfoViewController* infoScene = [segue destinationViewController];
    infoScene.note = self.photo.note;
}


@end

File: UISearch Demo/UISearch Demo/DetailViewController.swift
//
//  DetailViewController.swift
//  UISearch Demo
//
//  Created by Chang Tong Xue on 2015-12-31.
//  Copyright © 2015 DX. All rights reserved.
//

import UIKit

class DetailViewController: UIViewController {
    
    @IBOutlet weak var logo: UIImageView!
    var selectedTeam: Team!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        logo.image = UIImage(named: selectedTeam.image)
        navigationItem.title = selectedTeam.name
    }
    
}
